hil/cli/network.py

```diff
@@ -34,19 +36,64 @@ def network_delete(network):
 
 @network.command(name='show')
 @click.argument('network')
-def network_show(network):
+@click.option('--json', is_flag=True)
+def network_show(network, json):
     """Display information about network"""
     q = client.network.show(network)
-    for item in q.items():
-        sys.stdout.write("%s\t  :  %s\n" % (item[0], item[1]))
+    if json:
+        print q
+        return
+    net_table = PrettyTable()
+    net_table.field_names = ['Attribute', 'Info']
+
+    if 'owner' in q:
+        net_table.add_row(['owner', q['owner']])
+        net_table.add_row(['', ''])
+    if 'name' in q:
+        net_table.add_row(['name', q['name']])
+        net_table.add_row(['', ''])
+    if 'access' in q:
+        net_table.add_row(['access', q['access'][0]])
+        net_table.add_row(['', ''])
+    if 'channels' in q:
+        net_table.add_row(['channels', q['channels'][0]])
+        for i in range(1, len(q['channels'])):
+            net_table.add_row(['', q['channels'][i]])
+        net_table.add_row(['', ''])
+    if 'connected-nodes' in q:
+        firstElement = 0
+        for subVal in q['connected-nodes'].iteritems():
+            temp = []
+            temp.append(subVal[0])
+            temp.append(subVal[1][0])
+            if firstElement == 0:
+                net_table.add_row(['connected-nodes', "->".join(temp)])
+                firstElement += 1
+            else:
+                net_table.add_row(['', "->".join(temp)])
+    print net_table
 
 
 @network.command(name='list')
-def network_list():
+@click.option('--json', is_flag=True)
+def network_list(json):
     """List all networks"""
     q = client.network.list()
-    for item in q.items():
-        sys.stdout.write('%s \t : %s\n' % (item[0], item[1]))
+    if json:
+        print q
+        return
+    count = 0
+    net_table = PrettyTable()
+    net_table.field_names = ['network name', 'network id', 'project name']
+    for key1, value1 in q.iteritems():
+        for key2, value2 in value1.iteritems():
```

Comment:
could you use more descriptive variable names if possible? if not, some comments would be helpful.